tests/CMakeLists.txt

```diff
@@ -227,12 +228,12 @@ if(MFEM_MGIS_HAVE_TFEL)
         PROPERTY DEPENDS BehaviourTest)
     endif((CMAKE_HOST_WIN32) AND (NOT MSYS))
   endfunction(add_micromorphic_damage_3d_test)
-
-
   
   if(MFEM_USE_SUITESPARSE)
-	add_micromorphic_damage_2d_test2(2)
-#	add_micromorphic_damage_3d_test(2)
+  	message(STATUS "micromorphic_damage_2d_test2 has been disabled with SUITE SPARSE")
+        #add_micromorphic_damage_2d_test2(2)
+        message(STATUS "micromorphic_damage_3d has been disabled with SUITE SPARSE")
+	#add_micromorphic_damage_3d_test(2)
```

Comment:
```suggestion
  	message(STATUS "micromorphic_damage_2d_test2 has been disabled with SUITE SPARSE")
        message(STATUS "micromorphic_damage_3d has been disabled with SUITE SPARSE")
```

Reply:
Same comment:  "I'll soon be reactivating this test, so I'd rather keep it as a comment. We'll post a tag once this test and the next two will be reactivated."